File: sdk/src/main/java/org/p2p/solanaj/kits/MultipleAccountsInfo.java
package org.p2p.solanaj.kits;

import com.squareup.moshi.Json;

import org.p2p.solanaj.rpc.types.RpcResultObject;

import java.util.List;

public class MultipleAccountsInfo extends RpcResultObject {

    @Json(name = "value")
    private List<AccountInfoParsed> value;

    public List<AccountInfoParsed> getAccountsInfoParsed() {
        return value;
    }

    public void setAccountsInfoParsed(List<AccountInfoParsed> info) {
        value = info;
    }

}

File: solana/src/main/java/org/p2p/solanaj/kits/MultipleAccountsInfo.kt
package org.p2p.solanaj.kits

import com.google.gson.annotations.SerializedName
import org.p2p.solanaj.model.types.RpcResultObject

data class MultipleAccountsInfo(
    @SerializedName("value")
    var accountsInfoParsed: List<AccountInfoParsed>
) : RpcResultObject()
